chainerx_cc/chainerx/cuda/cuda_device/rounding.cu

```diff
@@ -0,0 +1,16 @@
+#include "chainerx/array.h"
```

Comment:
A nitpick but you can remove this include since it's not used.
```suggestion
```

Reply:
Fixed.